devicetypes/smartthings/aeotec-doorbell-siren-6.src/aeotec-doorbell-siren-6.groovy

```diff
@@ -244,13 +244,13 @@ def resetActiveSound() {
 		sendHubCommand(onOffCmd(0x00))
 	}
 	sendEvent([name: "alarm", value: "off"])
-	sendEvent([name: "chime", value: "off"])
 }
 
 def setActiveSound(soundId) {
 	String childDni = "${device.deviceNetworkId}:${soundId}"
 	def child = childDevices.find { it.deviceNetworkId == childDni }
-	child?.sendEvent([name: "chime", value: "chime"])
+	child?.sendEvent(name: "chime", value: "on", displayed: false)
+	child?.sendEvent(name: "alarm", value: "both", displayed: false)
```

Comment:
Was the `displayed: false` intentional?  If you do this, only the "off" events will ever show up.

Reply:
The `displayed: false` only makes sense on initialization, in `keepChildrenOnline()`, and probably for the parent events.